src/QirRuntime/build.py

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+# DEPRECATED !!!
```

Comment:
Is this script no longer meant to be used? If so, what is the reason to not just remove it?